DQMOffline/Configuration/test/BuildFile.xml

```diff
@@ -0,0 +1,5 @@
+<use   name="FWCore/Framework"/>
+<use   name="FWCore/Utilities"/>
+<bin file="TestRunner.cpp" name="TestDQMOfflineConfiguration">
+  <flags TEST_RUNNER_ARGS=" /bin/bash DQMOffline/Configuration/test runtests.sh"/>
```

Comment:
you can delete DQMOffline/Configuration/test/TestRunner.cpp and use the following to run the test. 
```
<test name="TestDQMOfflineConfiguration" command="runtests.sh --threads 1"/>
```
add `--threads` option to  cmsswSequenceInfo.py so that scram runtests always use 1 thread. Or if possible to break this tests in many then you can do something like

```
<test name="TestDQMOfflineConfiguration1" command="runtests.sh --threads 1 --workflows 1,2,3"/>
<test name="TestDQMOfflineConfiguration2" command="runtests.sh --threads 1 --workflows 4,5,6"/>
<test name="TestDQMOfflineConfiguration3" command="runtests.sh --threads 1 --workflows 7,8"/>
<test name="TestDQMOfflineConfiguration" command="runtests-final.sh">
  <flags PRE_TEST="TestDQMOfflineConfiguration1"/>
  <flags PRE_TEST="TestDQMOfflineConfiguration2"/>
  <flags PRE_TEST="TestDQMOfflineConfiguration3"/>
</test>
```
in this case scram can run TestDQMOfflineConfiguration1,2,3 in parallel.

Reply:
@smuzaffar thanks for the hint, I copied that pattern from other tests... (which probably also should not use it)

It should be pretty easy to break the test down into `DQM` and `VALIDATION`, more would get messy. I'll try that tomorrow.